dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -379,6 +379,11 @@ Path findClassLocation(String className) {
   private ImmutableSet<JavaClass> listClasses(Path jar) throws IOException {
     ImmutableSet.Builder<JavaClass> javaClasses = ImmutableSet.builder();
     for (String classFileName : listClassFileNames(jar)) {
+      if (classFileName.startsWith("META-INF.versions.")) {
+        // Linkage Checker does not support multi-release JAR (for Java 9+) yet
```

Comment:
add a link to the issue to support Java 9+ here

Reply:
Added issue URL one line below.